src/Core/Math/LinearAlgebra.inl

```diff
@@ -75,15 +75,15 @@ namespace Core
     }
 
     template <typename Vector_>
-    Scalar Vector::getNormAndNormalize(Vector_& v)
+    Scalar Vector::getNormAndNormalize( Vector_& v)

```

Comment:
If you really want to commit that change, you also want to add the space before the closing bracket.